math/perfect_number.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+
+using namespace std;
+
+void Check_Perfect_Number(int number, int i, int sum) {
```

Comment:
to be usable as a library function, Check_Perfect_Number shall not print the result but rather indicating it via a suitable return value. A boolean seems fine, I think. 

The function also depends on the paramter i and sum although this is not really needed. To keep cohesion low we could remove these parameters.